components/telemetry-operator/controller/logparser/controller.go

```diff
@@ -20,7 +20,8 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/prometheus/client_golang/prometheus"
+	commonmetrics "github.com/kyma-project/kyma/components/telemetry-operator/controller/metrics"
```

Comment:
Why 'commonmetrics'?  Just 'metrics' also fine?

Reply:
Just to be consistent with https://github.com/kyma-project/kyma/pull/15336/files#diff-2de746570890a034850c4581b8cb422d860cb434672d4a22b54d33ab6bffae4aR69. There `metrics` is already taken.

Reply:
Then let's name it controllermetrics